### DISCUSSION TITLE
nicer logo

### DISCUSSION THREAD

**Discussion by Famku** [jordan-dalby/ByteStash#50] [General]
First of all - nice app! 
My only question is, do you plan to create another logo for your app? 
I think this one looks like AI generated

**Comment by jordan-dalby** [comment]
It's AI inspired; I had a real person make it! Feel free to make a new one!

**Reply by Famku** [reply]
That wasn't a negative review.
Unfortunately, I'm not creative enough.
Maybe someone has a few suggestions

**Comment by EDIflyer** [comment]
I quite like the logo TBH - thought you'd appreciate the AI element given your DP, Famku 😉
